tests/kani/Intrinsics/Count/cttz_nonzero_panic.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// kani-verify-fail
+
+// Check that `cttz_nonzero` fails if zero is passed as an argument
+
+#![feature(core_intrinsics)]
+use std::intrinsics::cttz_nonzero;
+
+// Call `cttz_nonzero` with an unconstrained symbolic argument
+macro_rules! test_cttz_nonzero {
```

Comment:
Why is this named nonzero?

Reply:
Because it is testing the `cttz_nonzero` intrinsic.